README.md

```diff
@@ -0,0 +1,123 @@
+#### Part 1 - Short-answer questions.
+a) What does the command ‘git status’ output?</br>
+**It tells you what branch you're in and whether the repo is clean or whether there are uncommitted changes.**
+
+b) In SwiftUI, anything that gets drawn on the screen is a **View**.
+
+c) print(“Hello world”) is an example of a **function** call.
+viewModel.getData() is an example of a **method** call.
+
+d) Name some Views you have seen so far in SwiftUI.
+  - **body**
+  - **ContentView**
+  - **Text**
+  - **Button**
+  - **VStack**
+  - **HStack**
+  - **ZStack**
+  - **Slider**
+
+e) How do you create a new local repository using git? (Feel free to answer with how you use git, i.e. terminal or another app)</br>
+**In the GitHub app, press `command-n` or go to the File menu and select "new repository." In the dialogue box that appears, give your repository a name and description, and select a local path. You can also choose to add a readme file, a git ignore file, and a license. Then click "create repository."**
+
+f) How do you preview your app in multiple orientations?</br>
+**In Xcode, make sure the canvas is showing (`command-option-return`), and that the preview is running (`command-option-p`). In the bottom left corner of the canvas, click the variants button and select orientation variants from the dropdown menu. Then just click on the orientation you want. While previewing a given orientation, you can mouse over the left and right sides of the canvas and click the arrows that appear to switch between orientations.**
+
+g) An app is made up of **files** of **classes and structs** that contain **properties and methods**.
+
+h) Name two components of a SwiftUI Button.</br>
+**label and action**
+
+i) In git, what is the difference between a local repository and a remote repository?</br>
+**A local repository is stored on your local computer/hard drive, and a remote repository is stored on a server that you can access from anywhere using GitHub**
```

Comment:
A remote repository is located in the cloud or remote server. it may not be github

Reply:
Ah good to know, thanks.